tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, wait, createTestSelector, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { SettingsSelectors } from './settings.selectors';
+
+jest.setTimeout(60_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe.only(`Create and switch account`, () => {
```

Comment:
```suggestion
describe(`Create and switch account`, () => {
```

Reply:
My bad !

Reply:
Rather than using `.only()` I'm calling my tests directly

```
yarn jest --verbose=true --runInBand --testPathPattern=tests/integration/send-tokens.spec.ts
```

Reply:
Yes I do that do now, actually 'only' does not work somehow.

Reply:
Only is still here

Reply:
Fixed